tests/dummy/app/components/avatars-wrapper/template.hbs

```diff
@@ -6,4 +6,5 @@
   {{r-avatar src='avatars/1.png' size="x-large"}}
   {{r-avatar src='blah' fallbackSrc='avatars/default-pic.png' size="x-large"}}
   {{r-avatar fallbackSrc='avatars/default-pic.png' size="x-large"}}
+  {{r-avatar fallbackSrc='avatars/default-pic.png' sizeOverride='50' size="x-large"}}
```

Comment:
Can you add another sample that asserts that an avatar with no `size` prop but a `sizeOverride` prop still works? Other than that, this is good to go.